kivy/core/window/__init__.py

```diff
@@ -1431,6 +1433,9 @@ def on_resize(self, width, height):
         '''Event called when the window is resized.'''
         self.update_viewport()
 
+    def on_move(self):
+        self._moved = not self._moved
+
```

Comment:
Why do you need the `_moved` property? Could you not just instead directly dispatch `top`  and `left` here (`self.dispatch('top')`).